mobile/src/main/java/org/openhab/habdroid/ui/WidgetListFragment.kt

```diff
@@ -362,7 +360,10 @@ class WidgetListFragment :
             )
             nfcMenu.setHeaderTitle(R.string.item_picker_dialog_title)
             populateStatesMenu(nfcMenu, activity, suggestedCommands, suggestedCommands.shouldShowCustom) {
-                    state, mappedState, itemId ->
+                    state,
+                    mappedState,
+                    itemId
+                ->
```

Comment:
This looks somewhat ugly. Can the function parameter list be splitter over multiple lines instead?